### PR TITLE
Hockey app setup

### PR DESCRIPTION
**Pull request by Truhoan**
HockeyApp now has the android app published. 
I've tested that hockey actually sends out the published applications to selected users.

### CODE REVIEW

File: cykelplanen/build.gradle [hunk 1]
```diff
@@ -21,7 +21,7 @@ android {
         applicationId "com.spoiledmilk.cykelsuperstier"
         minSdkVersion 16
         targetSdkVersion 27
-        manifestPlaceholders = [HOCKEYAPP_APP_ID: ""]
+        manifestPlaceholders = [HOCKEYAPP_APP_ID: "cec6c0e595db43ffa1e9a60bfab16161"]
```
Comment (emiltin) [inline cykelplanen/build.gradle:24]: is this ok to have in the repo?

File: cykelplanen/build.gradle [hunk 2]
```diff
@@ -32,7 +32,7 @@ android {
     buildTypes {
         debug {
             signingConfig signingConfigs.development
-            manifestPlaceholders = [HOCKEYAPP_APP_ID: "bfcca234bd157f6eba719e4be7e09445"]
+            manifestPlaceholders = [HOCKEYAPP_APP_ID: "cec6c0e595db43ffa1e9a60bfab16161"]
```
Comment (emiltin) [inline cykelplanen/build.gradle:35]: same here

File: cykelplanen/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -2,10 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.spoiledmilk.cykelsuperstier"
-    android:versionCode="82"
-    android:versionName="2.2.3">
+    android:versionCode="83"
+    android:versionName="2.2.4">
 
-    <uses-sdk android:targetSdkVersion="23" />
+    <uses-sdk android:targetSdkVersion="27" />
```
Comment (emiltin) [inline cykelplanen/src/main/AndroidManifest.xml:8]: is this change required in this pr?

File: ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java [hunk 4]
```diff
@@ -114,78 +114,78 @@ public void launchMainMapActivity() {
...
+                startActivity(i);
+            } else if (IBikeApplication.isUserLogedIn()) {
+                Log.d("DV", "Prompting login for user!");
+                Intent i = new Intent(LoginSplashActivity.this, SignatureActivity.class).putExtra("normalUser", true).addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+                IBikeApplication.setWelcomeScreenSeen(true);
+                startActivity(i);
+            }
+        } else {
+            Log.d("DV", "We got a signature, enabling tracking!");
+            IBikeApplication.getSettings().setTrackingEnabled(true);
```
Comment (emiltin) [inline ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java:186]: not clear to me when this happens, and kind of tracking is turned on?

Comment (Truhoan) [reply ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java:186]: As commented on the other pull request, The tracking is disabled. It's part of the tracking project, that we discussed, we are not going to use.

File: cykelplanen/build.gradle (hunk 1 above)
Comment (Truhoan) [reply cykelplanen/build.gradle:24]: This is okay, to have in the repository. 
Is just a way for HockeyApp to recognize the app when uploaded. Only superusers in HockeyApp can upload.

File: ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java (hunk 4 above)
Comment (emiltin) [reply ibikecph-lib/src/main/java/com/spoiledmilk/ibikecph/login/LoginSplashActivity.java:186]: we'll clean it up later. as long as you're sure it's currently turned off. all this tracking should really have been done in a feature branch

File: cykelplanen/build.gradle (hunk 2 above)
Comment (Truhoan) [reply cykelplanen/build.gradle:35]: Same goes for this HockeyApp ID
